opg-php-fpm-1604/Dockerfile

```diff
@@ -32,12 +32,7 @@ RUN  sh /tmp/pecl_install.sh
 
 ADD  docker/nginx /etc/nginx
 
-RUN  pip install schedule statsd requests && \
-     curl -s https://raw.githubusercontent.com/ministryofjustice/php-fpm-stats-collector/master/php-fpm-stats-collector.py > /usr/local/bin/php-fpm-stats-collector.py && \
-     chmod a+x /usr/local/bin/php-fpm-stats-collector.py
-
-ADD  docker/service/php-fpm-stats /etc/sv/php-fpm-stats
-RUN  chmod a+x /etc/sv/php-fpm-stats/run && ln -s /etc/sv/php-fpm-stats /etc/service/
+RUN  pip install schedule statsd requests
```

Comment:
Are there any pip related cleanups that could be ran. Similar to how do purge composer cache or apt cache?

Reply:
Outside the scope of this ticket